tutorials/02-functions/02-what-is-this.md

```diff
@@ -137,9 +146,9 @@ Math.min.apply(null, numbers); // returns 2;
 
 ### A word on the spread operator in ES6/2015
 
-ES6 introduces the spread operator (`...`), which allows us to spread the contents of an array without using `apply()`. In the future, you'll be able to do the following:
+ES6 introduces the spread operator (`...`), which allows us to spread the contents of an array without using `apply()`. With ES6, you're able to do the following:
```

Comment:
I'm on the fence with this one because Safari still does not support the spread operator, but let's go with it.